<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CheckBoxsDialog</class>
 <widget class="QDialog" name="CheckBoxsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>801</width>
    <height>496</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(43, 125, 173);</string>
  </property>
  <widget class="QPushButton" name="OpenAllButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>40</y>
     <width>277</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>打开全部柜子</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ChekedButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>100</y>
     <width>277</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>已全部打开，进行检测</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AuthenButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>160</y>
     <width>277</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>确认认证</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ExitButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>220</y>
     <width>277</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>ExitButton</sender>
   <signal>clicked()</signal>
   <receiver>CheckBoxsDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>438</x>
     <y>439</y>
    </hint>
    <hint type="destinationlabel">
     <x>400</x>
     <y>247</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
